packages/network/src/createNetworkNode.ts

```diff
@@ -24,6 +24,9 @@ export interface NetworkNodeOptions extends AbstractNodeOptions {
     trackerConnectionMaintenanceInterval: number
     webrtcDisallowPrivateAddresses: boolean
     acceptProxyConnections: boolean
+    webrtcDatachannelMaxMessageSize?: number
+    webrtcDatachannelPortRangeBegin?: number
```

Comment:
To make the configuration easy to use, I'd maybe use only one parameter `webrtcPortRange` which would enclose the lower and upper bound into an object:
```
{
  min: number
  max: number
}
```

Most likely end-users would almost always define both the upper and lower bound? 

There are already quite many options for `createNetworkNode`, and therefore I'd like to keep the number of options very minimal. We may later add this same parameter to `streamr-client`, and also there is is better if we can just add one parameter instead of one. 

Also dropped `datachannel` from the name if this is datachannel is maybe a implementation specific detail.

(Noticed that we need to pass this to `node-datachannel` as two parameters. But that internal implementation is not relevant for end-users, and therefore we can just use one parameter here if we want.)